src/win_bindings.rs

```diff
@@ -0,0 +1,51 @@
+#![allow(unreachable_pub)]
+// Bindings generated by `windows-bindgen` 0.49.0
```

Comment:
Windows-bindgen inserts an empty line below the comment, that rustfmt removes. Maybe also something to open an issue for?